lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java

```diff
@@ -144,6 +147,25 @@ private void enablePassive(long interval) throws SecurityException {
     }
   }
 
+  private void checkLastKnownGps() {
+    checkLastKnownAndNotify(GPS_PROVIDER);
+  }
+
+  private void checkLastKnownNetwork() {
+    checkLastKnownAndNotify(NETWORK_PROVIDER);
+  }
+
+  private void checkLastKnownPassive() {
+    checkLastKnownAndNotify(PASSIVE_PROVIDER);
+  }
+
+  private void checkLastKnownAndNotify(String provider) {
```

Comment:
For location requests with `PRIORITY_HIGH_ACCURACY` this method would be invoked for both `GPS_PROVIDER` and `NETWORK_PROVIDER` resulting in two initial location updates.

We should only send one initial location update and use the logic in `FusionEngine.getLastLocation()` to decide which one is best.